src/groupeddataframe/utils.jl

```diff
@@ -5,11 +5,13 @@ function hashrows_col!(h::Vector{UInt},
                        v::AbstractVector{T},
                        rp::Nothing,
                        firstcol::Bool) where T
-    @inbounds for i in eachindex(h)
-        el = v[i]
-        h[i] = hash(el, h[i])
-        if length(n) > 0
```

Comment:
Can you remind me why did we do `length(n) > 0` check here? (is it that `length(n) > 0` if `skipmissing` is used?)

Reply:
Yes, we pass an empty vector when `skipmissing=false`.